commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -156,31 +262,35 @@ def poll_for_aws_state(env, instance_ids):
     }
     if not unfinished_instances:
         return {
-            instance['InstanceId']: instance['PublicIpAddress']
+            instance['InstanceId']: (instance['PublicIpAddress'], instance['PrivateIpAddress'])
             for instance in instances
             if instance['InstanceId'] in instance_ids
         }
 
 
 def save_inventory(env, inventory):
-    inventory_file_contents = yaml.safe_dump(inventory.to_json(), default_flow_style=False)
+    j2 = jinja2.Environment(loader=jinja2.FileSystemLoader(os.path.dirname(__file__)))
+    template = j2.get_template('inventory.ini.j2')
+    inventory_file_contents = template.render(inventory=inventory)
     inventory_file = get_inventory_filepath(env)
+    if not os.path.exists(os.path.dirname(inventory_file)):
+        os.makedirs(os.path.dirname(inventory_file))
     with open(inventory_file, 'w') as f:
         f.write(inventory_file_contents)
     print('inventory file saved to {}'.format(inventory_file),
           file=sys.stderr)
 
 
 def copy_default_vars(env, aws_config):
-    template_dir = os.path.join(VARS_DIR, 'dev')
+    vars_dir = VARS_DIR
+    template_dir = os.path.join(vars_dir, '.commcare-cloud-bootstrap')
     new_dir = ENVIRONMENTS_DIR
```

Comment:
What's going on with this guy?

Reply:
Hmm, yeah the place where it's used below in the comment doesn't really make sense anymore, and we no longer need to create this dir because it's already there from when we create it when we make the inventory. Also, I'm creating the wrong dir (`/environments` instead of `/environments/<env>`).

Reply:
I'll fix this in a followup PR. (Already changed this a bit in some secret new commits I've been working on.)